docs/eigenlayer/operator-guides/operator-installation.md

```diff
@@ -293,22 +291,23 @@ You must configure the correct DelegationManager contract address for your envir
 
 **Optional:** Set Delegation Approver
 
-Operators have the option to set [delegationApprover](https://github.com/Layr-Labs/eigenlayer-contracts/blob/dev/src/contracts/interfaces/IDelegationManager.sol#L30) when they register. This is address is used to approve delegation requests from stakers. The `delegationApprover` address will be required sign and approve new delegation from Stakers to this Operator. Otherwise, the `delegationApprover` will not be used if the default value is left as `0x000..`.
+Operators have the option to set [delegationApprover](https://github.com/Layr-Labs/eigenlayer-contracts/blob/dev/src/contracts/interfaces/IDelegationManager.sol#L30) when they register. This is address is used to approve delegation requests from stakers. The `delegationApprover` address will be required sign and approve new delegation from Stakers to this Operator. Otherwise, the `delegationApprover` will not be used if the default value is left as `0x000..`. Please see [delegationApprover Design Patterns](#delegationapprover-design-patterns) below for more detail.
```

Comment:
The current wording of this section has a few grammar problems and seems generally confusing to me.
Recommended edits:
`This address is used to approve delegation requests from stakers` (remove the 'is', although you could also realistically delete this entire sentence, since it is redundant with what immediately follows)
and
```
If the `delegationApprover` is set to a nonzero value, then the `delegationApprover` address will be required sign its approval of new delegations from Stakers to this Operator. If the default value is left as the zero address (0x000...), then all new delegations will be automatically approved without the need for any signature
```

Reply:
also going to recommend changing the link to the `mainnet` branch, instead of the `dev` branch.

Reply:
Good feedback! 

- grammar error fixed
- branch updated to mainnet
- updated to use the wording you suggested, it is more clear for the reader